qt/src/hocr/HOCRPdfExporter.cc

```diff
@@ -684,3 +712,43 @@ QImage HOCRPdfExporter::getSelection(const QRect& bbox)
 {
 	return m_displayerTool->getSelection(bbox);
 }
+
+void HOCRPdfExporter::paperSizeChanged() {
+	delete ui.lineEditPaperSizeHeight->validator();
+	delete ui.lineEditPaperSizeWidth->validator();
+
+	if(ui.comboBoxPaperSizeMetric->itemText(ui.comboBoxPaperSizeMetric->currentIndex()) == "mm") {
```

Comment:
Since "mm" and "inch" are localized, this breaks easily if the translated strings differ. => Don't use human strings for logic, rather use `enum class PaperSizeMetrics`.

Reply:
Fixed.